plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/internal/core/util/ResourceUtils.java

```diff
@@ -78,6 +78,23 @@ public Boolean visit(ITags capability) {
 		}, Boolean.FALSE);
 	}
 
+	public static boolean isMatching(final String filterText, String name, Collection<String> tags) {
+		if (StringUtils.isBlank(filterText)) {
```

Comment:
To me it looks as if this #isMatching (for tags) and the other #isMatching (for a resource) share at least
- entry parameter validation
- filterText sanitation (#replaceAll(",", " ")
- containsAll check.
Shouldnt we use share the code for better maintainability?